frontend/public/components/overview/build-overview.tsx

```diff
@@ -41,7 +44,7 @@ const BuildOverviewItem: React.SFC<BuildOverviewListItemProps> = ({ build }) =>
 
   const statusTitle = (
     <div>
-      Build <BuildNumberLink build={build} /> {conjugateBuildPhase(phase)}
+      {t('public~Build')} <BuildNumberLink build={build} /> {conjugateBuildPhase(phase, t)}
```

Comment:
I like minimal changes, but in this case we should transform two `t()` calls to a `<Trans` call. This will enable the i18n team to translate "Build #n was cancelled" as one string. Wdyt?